charts/stardog/values.yaml

```diff
@@ -32,6 +32,9 @@ serverStartArgs: ""
 # Stardog Java perf args, e.g. for changing Java gc
 stardogPerfJavaArgs: "-XX:SoftRefLRUPolicyMSPerMB=1 -XX:+UseG1GC -XX:+UseCompressedOops"
 
+# Optional base64 encoded Stardog License
```

Comment:
This will also need to be exposed in the installer UI.

Reply:
It is this is just the change on the helm chart for develop.